app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class SameNamesError(Exception):
+    pass
+
+
+class NotRightCommand(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        try:
+            command, source_file, destination_file = command.split()
+            if command != "cp":
+                raise NotRightCommand("Incorrect command")
+            if source_file == destination_file:
+                raise SameNamesError("File names are the same!")
```

Comment:
I recommend to replace this code by one "if" that will check all 3 conditions simultaneously and otherwise nothing happens